lib/reader.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require './modules/validation'
+class Reader
+  include Validation
+  attr_accessor :name, :email, :city, :street, :house
+
+  def initialize(name, email, address = {})
+    @name = name
+    @email = email
+    @city = address[:city]
+    @street = address[:street]
+    @house = address[:house]
+    validate(name, email, address)
+  end
+
+  def validate(name, email, address)
+    [name, email, address[:city], address[:street]].each do |item|
```

Comment:
Can we use instance variables for `city` and `street` instead of passing a key to the Hash?